modules/shared/src/main/scala/io/constellationnetwork/currency/dataApplication/package.scala

```diff
@@ -387,20 +390,43 @@ trait DataApplicationL1Service[F[_], D <: DataUpdate, DON <: DataOnChainState, D
     extends DataApplicationService[F, D, DON, DOF]
     with DataApplicationL1ContextualOps[F, D, DON, DOF] {
 
-  def postDataTransactionsRequestDecoder(req: Request[F])(implicit f: Async[F]): F[DataRequest] = {
+  def postDataTransactionsRequestDecoder(req: Request[F])(implicit F: Async[F]): F[DataRequest] = {
```

Comment:
I wonder if it is possible to define this implementation using the Client / Route structure defined in `node-shared`

one example like the Node interaction routes
- Client (for connecting from local to remote, i.e. Alice ) 
modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/http/p2p/clients/NodeClient.scala

- Route (for servicing local connection to remote)
modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/http/routes/NodeRoutes.scala